tests/Configurator/CopyFromPackageConfiguratorTest.php

```diff
@@ -58,7 +58,7 @@ public function testConfigureAndOverwriteFiles()
 
         $this->io->expects($this->at(0))->method('writeError')->with(['    Copying files from package']);
         $this->io->expects($this->at(2))->method('writeError')->with(['      Created <fg=green>"./public/file"</>']);
-        $this->io->method('askConfirmation')->with('File "build/public/file" has uncommitted changes, overwrite? [y/N] ')->willReturn(true);
+        $this->io->method('askConfirmation')->with('File "file" has uncommitted changes, overwrite? [y/N] ')->willReturn(true);
```

Comment:
why did this pass before and it does now?
wasn't the previous path better for the purpose?

Reply:
I don't know, I can't explain it, could it be that the git output has changed?
Indeed, the previous path is better